dockerbuild/config/config.php

```diff
@@ -915,7 +915,7 @@
      */
     'language.available' => [
         'en', 'no', 'nn', 'se', 'da', 'de', 'sv', 'fi', 'es', 'ca', 'fr', 'it', 'nl', 'lb',
-        'cs', 'sk', 'sl', 'lt', 'hr', 'hu', 'pl', 'pt', 'pt-br', 'tr', 'ja', 'zh', 'zh-tw',
+        'cs', 'sk', 'sl', 'lt', 'hr', 'hu', 'pl', 'pt', 'pt_BR', 'tr', 'ja', 'zh', 'zh_TW',
```

Comment:
Was this the only change in the config.php.dist template? I'm just wondering how you found this. If possible, do a diff against the [config.php.dist](https://github.com/simplesamlphp/simplesamlphp/blob/master/config/config.php.dist) file and update the comment at the top of this file. Also, please do the same for [UserPass.php](https://github.com/sil-org/ssp-base/blob/97e41537a546c5b36c9a59aa0ef648e962db81b7/development/UserPass.php#L5).

Reply:
Updated `config.php`. There are no changes needed for `UserPass.php`, would you like me to comment on that anyway?